src/modules/link-tooltip.coffee

```diff
@@ -82,6 +84,7 @@ class LinkTooltip extends Tooltip
         @textbox.setSelectionRange(0, url.length)
       )
     else
+      dom(@container).addClass('open')
```

Comment:
Instead of adding the `open` class here, I think doing it in `show()` might be a better catch-all place? https://github.com/voxmedia/quill/blob/link-tooltip-open-state/src/modules/tooltip.coffee#L65